src/App.js

```diff
@@ -1,16 +1,130 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import axios from "axios";
+import Graph from "./Graph"
 
 class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      userInput: "",
+      currCity: "",
+      currTemp: "",
+      weatherType: "",
+      weatherDesc: "",
+      weatherIconCode: "",
+      forecast: false,
+      forecastData: [],
+    };
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleCheck = this.handleCheck.bind(this)
+  }
+
+  handleChange = (e) => {
+    this.setState({userInput: e.target.value})
+  }
+
+  handleCheck = () => {
+    this.setState({forecast: !this.state.forecast})
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .get(
+        `https://api.openweathermap.org/geo/1.0/direct?q=${this.state.userInput}&limit=1&appid=${process.env.REACT_APP_WEATHER_API_KEY}`
+      )
+      .then((response) => response.data[0])
+      .then((cityGeoData) =>
+        axios.get(
+          `https://api.openweathermap.org/data/2.5/weather?lat=${cityGeoData.lat}&lon=${cityGeoData.lon}&units=metric&appid=${process.env.REACT_APP_WEATHER_API_KEY}&units=metric`
+        )
+      )
```

Comment:
As discussed, can also skip the second .then like so. Matter of style/choice :)!
```suggestion
      .then((response) => { 
                 const cityGeoData = response.data[0]
                 return axios.get(
          `https://api.openweathermap.org/data/2.5/weather?lat=${cityGeoData.lat}&lon=${cityGeoData.lon}&units=metric&appid=${process.env.REACT_APP_WEATHER_API_KEY}&units=metric`
        )
       })
```